CMakeLists.txt

```diff
@@ -355,7 +356,9 @@ endif()
 # =============================================================================
 # Install targets
 # =============================================================================
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/share/lib DESTINATION ${NRN_INSTALL_SHARE_DIR})
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/share/lib
+        DESTINATION ${NRN_INSTALL_SHARE_DIR}
+        PATTERN "python" EXCLUDE)
```

Comment:
This is a great idea. Useless to have two modules of the same thing installed. And this one is broken because of the missing shared libraries created by setup.exe. However I wonder if nrn/src/nrnpython/nrnpython.cpp needs to be tweaked now as it may add this folder to the sys.path.

Reply:
> However I wonder if nrn/src/nrnpython/nrnpython.cpp needs to be tweaked now as it may add this folder to the sys.path.

The fact that the CI passes on normal builds with Python, doesnt ensure things are not broken? Or you fear for the case in Windows?

Reply:
As this has nothing to do with wheels and as a non-existent folder in sys.path is not an error,
this can be reviewed as its own separate issue.